### PR TITLE
Possible to change status bar color & preview image quality

### PR DESCRIPTION
**Pull request by pak-amsiq**
SwiftLint should be alright 👍

### CODE REVIEW

File: GBHFacebookImagePicker/Classes/Controller/GBHFacebookAlbumPicker.swift [hunk 1]
```diff
@@ -54,6 +59,9 @@ class GBHFacebookAlbumPicker: UITableViewController {
 
         // Start Facebook login
         self.doFacebookLogin()
+
+        // Statusbar
+        self.setNeedsStatusBarAppearanceUpdate()
```
Comment (floriangbh) [inline GBHFacebookImagePicker/Classes/Controller/GBHFacebookAlbumPicker.swift]: I don't think this is useful. Maybe it's only required when you change a status bar propreties when the controller is already visible ?

File: GBHFacebookImagePicker/Classes/Controller/GBHPhotoPickerViewController.swift [hunk 2]
```diff
@@ -80,6 +85,9 @@ class GBHPhotoPickerViewController: UIViewController {
 
         // Fetch photos if empty
         self.getPhotos()
+
+        // Statusbar
+        self.setNeedsStatusBarAppearanceUpdate()
```
Comment (floriangbh) [inline GBHFacebookImagePicker/Classes/Controller/GBHPhotoPickerViewController.swift]: I don't think this is useful. Maybe it's only required when you change a status bar propreties when the controller is already visible ?

**Comment by floriangbh** [comment]
Thanks :-)

**Comment by floriangbh** [comment]
Your last commit is "In progress" ? 
Can't merge until it's finish :-)

File: GBHFacebookImagePicker/Classes/Controller/GBHFacebookAlbumPicker.swift (hunk 1 above)
Comment (pak-amsiq) [reply GBHFacebookImagePicker/Classes/Controller/GBHFacebookAlbumPicker.swift]: You are right. I just removed these lines

File: GBHFacebookImagePicker/Classes/Controller/GBHPhotoPickerViewController.swift (hunk 2 above)
Comment (pak-amsiq) [reply GBHFacebookImagePicker/Classes/Controller/GBHPhotoPickerViewController.swift]: You are right. I just removed these lines

**Comment by pak-amsiq** [comment]
Wasn't on purpose that the last commit was uploaded for this PR, as it isn't ready atm. 

These two commits should be ready for merging, with your latest comments included :-)

Please make a release as soon as possible :-)

**Pull request merged**